src/main/java/edu/wpi/cs3733/D22/teamC/entity/service_request/lab_system/LabSystemSRDAOImpl.java

```diff
@@ -126,7 +126,7 @@ public boolean updateServiceRequest(LabSystemSR serviceRequest) {
                 if (successParent) {
                     // Update the child-unique attributes in the child table.
                     PreparedStatement statement = DBManager.getInstance().connection.prepareStatement(
-                            "UPDATE LAB_SYSTEM_SR SET labType = ?, patientID = ? " +
+                            "UPDATE LAB_SYSTEM_SR SET LabType = ?, PatientID = ? " +
```

Comment:
Be careful with unassociated chances on your PRs, could cause merge conflicts with the LabSystemSRDao ticket.